examples/wc-dapp-react/src/components/HelloWorld.tsx

```diff
@@ -532,6 +533,14 @@ function HelloWorld() {
                 Wipe Methods
               </button>
               <br></br>
+              Dapp method invocation context (optional):
+              <br></br>
+              <textarea
+                data-testid="hello-world__contextual_message"
+                placeholder="Write a context message for method invocation"
+                onChange={(event) => setContext(event.target.value)}
+              />
+              <br></br>
```

Comment:
I don't think we need this, let's keep the examples simple, let's just use an hardcoded context message